content/pt/docs/concepts/configuration/organize-cluster-access-kubeconfig.md

```diff
@@ -0,0 +1,131 @@
+
+   Caso contrário, se a variável de ambiente `KUBECONFIG` estiver definida, esta deverá ser utilizada como uma lista de arquivos a serem combinados.
+   Combine os arquivos listados na variável de ambiente `KUBECONFIG` de acordo com as seguintes regras:
+
+    * Ignorar arquivos vazios.
+    * Produzir erros para aquivos cujo conteúdo não for possível desserializar. 
+    * O primeiro arquivo que definir um valor ou mapear uma chave determinada, será o escolhido.
+    * Nunca modificar um valor ou mapear uma chave.
+	   Exemplo: Preservar o contexto do primeiro arquivo que definir `current-context`.
+		 Exemplo: Se dois arquivos especificar um `red-user`, use apenas os valores do primeiro `red user`. Mesmo se um segundo arquivo possuir entradas não conflitantes sobre a mesma entrada `red-user`, estas deverão ser descartadas.
+
+   Para um exemplo de definição da variável de ambiente `KUBECONFIG` veja [Definido a variável de ambiente KUBECONFIG](/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable).
+
+   Caso contrário, utilize o arquivo kubeconfig padrão encontrado no diretório `$HOME/.kube/config`, sem qualquer tipo de combinação.
+
+1. Determine o contexto a ser utilizado baseado no primeiro padrão encontrado, nesta ordem: 
+
+    1. Usar o conteúdo da flag `--context` caso ela existir.
+    1. Usar o `current-context` a partir da combinação dos arquivos kubeconfig.
+
+   Um contexto vazio é permitido neste momento.
+
+
+1. Determinar o cluster e o usuário. Neste ponto, poderá ou não existir um contexto.
+   Determinar o cluster e o usuário no primeiro padrão encontrado de acordo com a ordem à seguir. Este procedimento deverá executado duas vezes: uma para definir o usuário a outra para definir o cluster.
+
+	1. Utilizar a flag caso ela existir: `--user` ou `--cluster`.
+	1. Se o contexto não estiver vazio, utilizar o cluster ou usuário deste contexto.
+
+   O usuário e o cluster poderá estar vazio neste ponto.
+
+1. Determinar as informações do cluster atual a serem utilizadas. Neste ponto, poderá ou não existir informações de um cluster.
+
+   Construir cada peça de informação do cluster baseado nas opções à seguir; a primeira ocorrência encontrada será a opção vencedora: 
+
+  1. Usar as flags de linha de comando caso existirem: `--server`, `--certificate-authority`, `--insecure-skip-tls-verify`.
+  1. Se algum atributo do cluster existir a partir da combinação de kubeconfigs, estes deverão ser utilizados.
+  1. Se não existir informação de localização do servidor falhar. 
+
+1. Determinar a informação atual de usuário a ser utilizada. Construir a informação de usuário utilizando as mesmas regras utilizadas para o caso de informações de cluster, exceto para a regra de técnica de autenticação que deverá ser única por usuário:
+   
+	1. Usar as flags, caso existirem: `--client-certificate`, `--client-key`, `--username`, `--password`, `--token`.
```

Comment:
missing space here, this is a subitem from the previous item (see the preview vs https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/, items are numbered incorrectly